### PR TITLE
[Doc] user manual updates for 3.4.9 release

### PR DESCRIPTION
**Pull request by carlaschroder**
Please review @rperezb @nasli and whoever else wants to join the party :)

### CODE REVIEW

File: user_manual/ios_app.rst [hunk 1]
```diff
@@ -33,10 +32,10 @@ Managing Your Files
 
 All your files on your ownCloud server are listed on the Files page, but are not 
 downloaded to your iPhone or iPad until you tap on them. Downloaded files are 
-marked with a little green arrow. Image files display thumbnails after they have 
-been downloaded; image files that have not been downloaded display generic 
-icons. Click the plus button at the top right to upload files to your ownCloud 
-server, or to create a new folder. 
+marked with a little green arrow. Thumbnails are displayed for both 
+downloaded and not-downloaded files. Click the overflow button at the top right
```
Comment (rperezb) [inline user_manual/ios_app.rst]: Non-downloaded thumbnails relies on the server, from oc8 and on

**Comment by rperezb** [comment]
@carlaschroder nice!

File: user_manual/ios_app.rst [hunk 2]
```diff
@@ -71,11 +73,12 @@ files, click the Share button to see who you have shared with, and to remove
...
-files with ownCloud 9.x users, you can enter their username and server URL in 
-the User and Groups dialog to create a direct share; for example 
-freda@example.com/owncloud. Your recipient will receive a notification asking 
-if they wish to accept the share, and then all they have to do is click an 
-"Accept" button.
+files with ownCloud 9.x users, you can enter their username (auto-completion is 
+supported for Federation sharing) and server URL in the User and Groups dialog 
+to create a direct share; for example freda@example.com/owncloud. (The exact 
+Federation share link is on every ownCloud user's Personal page in the ownCloud 
+Web interface.) Your recipient will receive a notification asking if they wish
```
Comment (mRoRo) [inline user_manual/ios_app.rst]: I'd somehow clarify that the notification is available in the web interface. It's not implemented in the iOS app till the moment.

File: user_manual/ios_app.rst (hunk 1 above)
Comment (carlaschroder) [inline user_manual/ios_app.rst]: lol, we'll probably have oC 7 users for years to come.

File: user_manual/ios_app.rst (hunk 2 above)
Comment (carlaschroder) [inline user_manual/ios_app.rst]: good, thanks.

Comment (carlaschroder) [inline user_manual/ios_app.rst]: @mRoRo What happens in the iOS app? Does the recipient of the federation share have to use the Web interface to accept the new share?

Comment (rperezb) [inline user_manual/ios_app.rst]: @carlaschroder for the moment yes, it's still not implemented the option to accept federated
@MTRichards  ^_^

Comment (carlaschroder) [inline user_manual/ios_app.rst]: thanks :)

Comment (carlaschroder) [inline user_manual/ios_app.rst]: It would also be cool to see the user's Federation share URL in the app, just like on the Personal page in the Web GUI.

**Comment by carlaschroder** [comment]
New commit with corrections! \o/

**Pull request merged**